sdk/python/feast/infra/offline_stores/redshift_source.py

```diff
@@ -372,3 +376,28 @@ def to_proto(self) -> SavedDatasetStorageProto:
 
     def to_data_source(self) -> DataSource:
         return RedshiftSource(table=self.redshift_options.table)
+
+
+class RedshiftLoggingDestination(LoggingDestination):
+    _proto_attr_name = "redshift_destination"
+
+    table: str
+
+    def __init__(self, table_ref: str):
+        self.table = table_ref
+
+    @classmethod
+    def from_proto(cls, config_proto: LoggingConfigProto) -> "LoggingDestination":
+        return RedshiftLoggingDestination(
+            table_ref=config_proto.redshift_destination.table_ref,
+        )
+
+    def to_proto(self) -> LoggingConfigProto:
+        return LoggingConfigProto(
+            redshift_destination=LoggingConfigProto.RedshiftDestination(
+                table_ref=self.table
+            )
+        )
+
+    def to_data_source(self) -> DataSource:
+        return RedshiftSource(table=self.table)
```

Comment:
I think you also need to pass `schema` (optional which defaults to `public` in RedshiftSource). Confusing name, but schemas as like namespaces / databases. If you don't specify schema, we generally use `public.table_name`, but you can override schema and say get `my_schema.table_name` which is different from `public.table_name`.

The idea is that user may want to use non-public Redshift schema for writing Feast logs.

Reply:
There was comment from @adchia suggesting that we should rather use schema from store configuration. See https://github.com/feast-dev/feast/pull/2574#discussion_r854413831